changelog/16522.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:change
```

Comment:
I think you need to move to the [correct changelog format](https://github.com/hashicorp/vault/blob/main/changelog/_go-ver-1120.txt).

Reply:
hmm. I don't see any differences other than the name?

Reply:
Yeah, the name is different, that's why the CI `Check Changelog` pass failed :-) 

(See the internal release process confluence for some discussion of the naming structure for Go version bumps in particular).

Reply:
ah, I see: `Please use the format changelog/go-ver-<VAULT_VERSION_WITHOUT_DOTS>.txt instead`

Reply:
Oh hm, it needs an underscore in the example. Could've sworn I saw a PR from @mladlow to fix that that @stevendpclark merged, but now I don't see it, hmmmmmm

Reply:
Done, though it is a bit confusing since there is `_go-ver-1120.txt` and now `go-ver-1120.txt`

Reply:
Yeah, its so we only have one Go version number in the final release. :-)

Reply:
Makes sense

Reply:
Oops I missed the fact that it was done on vault-enterprise during my review *sigh*. We will need to bring this to OSS.

https://github.com/hashicorp/vault-enterprise/pull/3050/files